ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -0,0 +1,91 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  policy_maps:
+    type: dict
+    display_name: QOS Policy-map
```

Comment:
I appreciate that the name comes from the current AVD docs, but given that policy-maps are used for QoS and PBR, should we just refer to this as a Policy-Map and not a QOS Policy-Map?

Reply:
so, as it will be similar to the key, we can remove display_name here? @ClausHolbechArista

Reply:
@Shivani-chourasiya We want the dash, so let's just write `display_name: Policy-Map`

Reply:
Updated